robot_smach_states/src/robot_smach_states/manipulation/grasp_point_determination.py

```diff
@@ -34,6 +34,24 @@ def get_grasp_pose(self, entity, arm):
         :param arm: arm to use
         :return: FrameStamped with grasp pose in map frame
         """
+        try:
+            # Try to compute the grasp pose based on the entity
+            return self._get_grasp_pose(entity, arm)
+        # noinspection PyBroadException
+        except Exception as e:
+            rospy.logwarn("Cannot compute grasp pose, defaulting to entity position and "
+                          "orientation the same as the robot")
+            robot_pose_map = self._robot.base.get_location()  # type: FrameStamped
+            return FrameStamped(kdl.Frame(robot_pose_map.frame.M, entity.pose.frame.p), entity.frame_id)
```

Comment:
The frames of the robot pose and entity pose may not match

Reply:
Good point. I only changed this in order to get my script to work. To validate this assumption I added a check: 9e12e56a5e4b0ec4ce0b364245bab356b8fd6181

Further improvements are IMHO out of scope

Reply:
Agreed